docs/06_how-to-guides/40_multi-index/how-to-define-a-singleton.md

```diff
@@ -112,3 +147,13 @@ __singleton_example.cpp__
 
 [[info | Full example location]]
 | A full example project demonstrating the instantiation and usage of singleton can be found [here](https://github.com/EOSIO/eosio.cdt/tree/master/examples/singleton_example).
+
+## Summary
+
+In conclusion, the above instructions show how to define a singleton.
+
+## Next Steps
+
+The following option is available when you complete the procedure:
+
+* Because a singleton is using as underlying structure a multi-index table, you can [iterate and retrieve data](./how-to-iterate-and-retrieve-a-multi_index-table) from the singleton same as you would with a multi-index table.
```

Comment:
Because a singleton is using as underlying structure a multi-index table, you can [iterate and retrieve data](./how-to-iterate-and-retrieve-a-multi_index-table) from the singleton same as you would with a multi-index table.

Shouldn't start with "Because"
maybe change to ->
How to  [iterate and retrieve data](./how-to-iterate-and-retrieve-a-multi_index-table) from the singleton.